### PR TITLE
chore: update source configuration response test data

### PR DESCRIPTION
**Pull request by saikumarrs**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Linear task (optional)

https://linear.app/rudderstack/issue/SDK-661/update-source-configuration-service-to-add-a-new-root-level-field-for

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] IE11

## Sanity Suite

- [ ] All sanity suite test cases pass locally

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: packages/sanity-suite/__fixtures__/sourceConfig1.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 {
+  "consentManagementMetadata": {
```
Comment (bardisg) [inline packages/sanity-suite/__fixtures__/sourceConfig1.json:2]: this will break the tests untill the updated sourceConfig response is deployed right?

Comment (saikumarrs) [reply packages/sanity-suite/__fixtures__/sourceConfig1.json:2]: Yes, but we can merge and deploy only after the config backend service is deployed.

Comment (bardisg) [reply packages/sanity-suite/__fixtures__/sourceConfig1.json:2]: perfect

**Review by bardisg — APPROVED** [review]

**Comment by saikumarrs** [comment]
Do not merge the PR until the config BE service release is done.

**Pull request merged**